### PR TITLE
Improve the user experience

### PR DESCRIPTION
**Pull request by mchv**
- Implement proper templating
- rework a bit on the timeline to be more intuitive
- add a new header based on @na7hangood

screen shot 2017-02-23 at 11 24 12
screen shot 2017-02-23 at 11 24 23
screen shot 2017-02-23 at 11 24 42

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
👍  🎨

File: src/templates/main.html [hunk 1]
```diff
@@ -1,72 +1,40 @@
...
 
-<p>Chuka Umanna, Labour Party </p>
+<p> {{who}}</p>
 
-<h2> Related articles </h2>
+<h2>Related articles</h2>
 <ul>
-    <li><a href=""> plop </a></li>
-    <li><a href=""> plop </a></li>
+    {{#each articles}}
```
Comment (LATaylor-guardian) [inline src/templates/main.html]: What is the templating library used here?

Comment (mchv) [reply src/templates/main.html]: [handlebar.js](http://handlebarsjs.com/)  :wavy_dash:

Comment (LATaylor-guardian) [reply src/templates/main.html]: Such a hipster! 😆

**Comment by mchv** [comment]
Few improvements based on @na7hangood feedback:
- Grouped articles by category
- Rename `Related articles` to `Find out more`

screen shot 2017-02-23 at 12 06 13

File: src/templates/main.html (hunk 1 above)
Comment (mchv) [reply src/templates/main.html]: @LATaylor-guardian I am thinking of buying some oil from @oilnam !

Comment (oilnam) [reply src/templates/main.html]: Oh dear 😅

**Pull request merged**